Description: Change the install prefix
 Upstream has a peculiar way of writing the install directive in the Makefile.
 I changed it to allow the package build to work properly, which included
 changing the prefix from /usr/local to /usr.
Author: Peter J. Mello <admin@petermello.net>
Origin: vendor
Forwarded: no
Last-Update: 2019-05-14

--- a/Makefile
+++ b/Makefile
@@ -8,11 +8,8 @@ ifeq ($(UNAME_S),FreeBSD)
 	OBJS += platform/freebsd/freebsd_ptrace.o platform/freebsd/freebsd.o
 	LDFLAGS += -lprocstat
 endif
-# Note that because of how Make works, this can be overridden from the
-# command-line.
-#
-# e.g. install to /usr with `make PREFIX=/usr`
-PREFIX=/usr/local
+
+PREFIX=/usr
 BINDIR=$(PREFIX)/bin
 MANDIR=$(PREFIX)/share/man

@@ -46,15 +43,16 @@ clean:
 BASHCOMPDIR ?= $(shell $(PKG_CONFIG) --variable=completionsdir bash-completion 2>/dev/null)

 install: reptyr
-	install -d -m 755 $(DESTDIR)$(BINDIR)
-	install -m 755 reptyr $(DESTDIR)$(BINDIR)/reptyr
-	install -d -m 755 $(DESTDIR)$(MANDIR)/man1
-	install -m 644 reptyr.1 $(DESTDIR)$(MANDIR)/man1/reptyr.1
-	install -d -m 755 $(DESTDIR)$(MANDIR)/fr/man1
-	install -m 644 reptyr.fr.1 $(DESTDIR)$(MANDIR)/fr/man1/reptyr.1
+	install -d -m 0755 $(DESTDIR)$(BINDIR)
+	install -D -m 0755 reptyr $(DESTDIR)$(BINDIR)/reptyr
+	install -d -m 0755 $(DESTDIR)$(MANDIR)/man1
+	install -D -m 0644 reptyr.1 $(DESTDIR)$(MANDIR)/man1/reptyr.1
+	install -d -m 0755 $(DESTDIR)$(MANDIR)/fr/man1
+	install -D -m 0644 reptyr.fr.1 $(DESTDIR)$(MANDIR)/fr/man1/reptyr.1
 	bashcompdir=$(BASHCOMPDIR) ; \
 	test -z "$$bashcompdir" && bashcompdir=/etc/bash_completion.d ; \
-	install -d -m 755 $(DESTDIR)$$bashcompdir ; \
-	install -m 644 reptyr.bash $(DESTDIR)$$bashcompdir/reptyr
+	install -d -m 0755 $(DESTDIR)$$bashcompdir ; \
+	install -D -m 0644 reptyr.bash $(DESTDIR)$$bashcompdir/reptyr

 .PHONY: PHONY
+
